src/open_r1/rewards.py

```diff
@@ -492,3 +499,33 @@ async def run_script(sbx: AsyncSandbox, script: str, language: str) -> float:
         return float(execution.text)
     except (TypeError, ValueError):
         return 0.0
+
+
+def get_reward_funcs(script_args: "GRPOScriptArguments") -> list[Callable]:
```

Comment:
Would it make sense to unit test this just to sanity check we are getting the expected reward functions?

Reply:
good point, I will add this.